examples/Workshop/Workshop.ino

```diff
@@ -6,16 +6,16 @@
 // After you registered your ABP device, go to The Things Network Dashboard
 // and copy the Device Addr, Network Session Key and App Session Key
 
-// Set your Device Address, for example: { 0x02, 0xDE, 0xAE, 0x00 };
-const byte devAddr[4] = { ... };
+// Set your Device Address, for example: "1C55B133";
+const String devAddr = " ... ";
```

Comment:
While you're at it, just change these constant placeholders to be like the other examples